### PR TITLE
Run test cases only if the extension is installed

### PR DESCRIPTION
**Pull request by hitesh96db**
I forgot to add this in the previous PR.

### CODE REVIEW

File: test/selenium/specs/ts_all_specs.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 require_relative "tc_history"
 # Don't run the posting process tests on Chrome as the
 # posting process has changed.
-if @browser == "firefox" || @browser == "safari"
+if @browser == "firefox"
```
Comment (hitesh96db) [inline test/selenium/specs/ts_all_specs.rb]: Also, I assumed that these same tests would pass for Safari, but there seems to be an issue when these tests run on Safari. @sammyshj will look into it.

Comment (sammyshj) [inline test/selenium/specs/ts_all_specs.rb]: Thanks for adding this!

**Comment by smcgregor** [comment]
I think a better fix here is to exclude cases where the extension is not installed.

@sammyshj: Does the test pass for safari_extension?

**Comment by hitesh96db** [comment]
@smcgregor Pushed new changes.

**Comment by sammyshj** [comment]
@smcgregor I am working on getting the safari_extension tests to pass, but, as mentioned earlier, there seems to be a problem with Selenium not being able to load the complete page when navigated to an extension page in Safari. Can you suggest how I should go ahead with this?

**Comment by smcgregor** [comment]
I suspect the Selenium extension is sandboxed from our extension. If they are both packaged into the same extension build, then they should not be sandboxed. I recommend looking into the Selenium code base to assess feasibility.

**Pull request merged**